### PR TITLE
Collijk/feature/vaccine updates

### PR DESCRIPTION
**Pull request by collijk**
Sorry this is a big messy PR.  It's unifying three attempts at this from two people as requirements were changing.  

Nearly all the work of interest is in beta_forecast.py and the modules it calls functions from: ode_forecast.py and forecast_metrics.py.

### CODE REVIEW

File: specification_templates/forecast.yaml [hunk 1]
```diff
@@ -4,9 +4,9 @@ data:
   output_root: ''
 scenarios:
   worse:
-    algorithm: 'normal' # normal, mean_level_mandate_reimposition, draw_level_mandate_reimposition
```
Comment (collijk) [inline specification_templates/forecast.yaml:7]: Removed all code related to mean level mandate reimposition.

File: specification_templates/forecast.yaml [hunk 2]
```diff
@@ -4,9 +4,9 @@ data:
...
+    algorithm: 'normal' # normal, draw_level_mandate_reimposition
     solver: 'RK45'
-    system: 'new_theta'
```
Comment (collijk) [inline specification_templates/forecast.yaml:9]: Removed 'old_theta' now that 'new_theta' has been in prod for a couple of months.

File: specification_templates/vaccine.yaml [hunk 3]
```diff
@@ -1,19 +1,19 @@
 data:
-  regression_version: '2020_09_15.01'
-  covariate_version: '2020_09_10.03'
+  regression_version: '2020_11_15.02'
+  covariate_version: '2020_11_15.02'
```
Comment (kathrynline) [inline specification_templates/vaccine.yaml:3]: I'm obviously not an expert on this codebase. Are all of these yaml files hand-entered?

Comment (collijk) [reply specification_templates/vaccine.yaml:3]: Yes.  In lieu of having about 30 cli args, the options are all specified here.

File: src/covid_model_seiir_pipeline/forecasting/task/beta_forecast.py [hunk 4]
```diff
@@ -23,21 +23,38 @@ def run_beta_forecast(draw_id: int, forecast_version: str, scenario_name: str, *
...
+    # We'll use the same params in the ODE forecast as we did in the fit.
+    beta_params = data_interface.load_beta_params(draw_id=draw_id)
     # Thetas are a parameter generated from assumption or OOS predictive
     # validity testing to curtail some of the bad behavior of the model.
     thetas = data_interface.load_thetas(scenario_spec.theta)
+    if ((1 < thetas) | thetas < -1).any():
+        raise ValueError('Theta must be between -1 and 1.')
+    if (beta_params['sigma'] - thetas >= 1).any():
+        raise ValueError('Sigma - theta must be smaller than 1')
+
```
Comment (miker985) [inline src/covid_model_seiir_pipeline/forecasting/task/beta_forecast.py:35]: I'm sure I've seen this check before. Any idea if that PR supersedes this or if it's duplicated logic?

Comment (collijk) [reply src/covid_model_seiir_pipeline/forecasting/task/beta_forecast.py:35]: This just moves the check

**Review by miker985 — APPROVED** [review]
Gentle request to method parameter lists in how you define and call them.

Basically just 1 arg per line and always include trailing commas for the last arg.

You do this is many places but not always, and it creates a lot of noise when things are being updated.

**Pull request merged**